### PR TITLE
Bullet update optional

### PR DESCRIPTION
**Pull request by derkachdaniil**
### Brief description of the changes:
putting values from physical-projectile-damage functions to tables.lua. highly optional branch of bullet_update_3(PR#103)

### Tested Changes:
- [ ] I've tested the changes locally or with people.
- [x] I've not tested the changes.

### CODE REVIEW

File: maps/biter_battles_v2/functions.lua [hunk 1]
```diff
@@ -146,7 +172,6 @@ function Public.combat_balance(event)
 		if balance_functions[key] then
 			if not global.combat_balance[force_name] then global.combat_balance[force_name] = {} end
 			balance_functions[key](force_name)
-			return
```
Comment (kajendra) [inline maps/biter_battles_v2/functions.lua:149]: Removing this return value will cause the substring to continue?
Is this intentional?

File: maps/biter_battles_v2/functions.lua [hunk 2]
```diff
@@ -12,6 +12,11 @@ local string_find = string.find
 
 -- Only add upgrade research balancing logic in this section
 -- All values should be in tables.lua
+local function proj_buff(current_value,force_name)
```
Comment (kajendra) [inline maps/biter_battles_v2/functions.lua:15]: I wasn't sure what this function was for at first. From what I understand, you are making sure we don't add 28 lines of code when you can create a separate method for the upgrade value!

We should at some point refactor the rest of the code to somehow do that...

A note, should the `current_value` variable be named `upgrade_value`?

**Review by kajendra — COMMENTED** [review]
Just a few things to clarify here

File: maps/biter_battles_v2/functions.lua (hunk 1 above)
Comment (derkachdaniil) [reply maps/biter_battles_v2/functions.lua:149]: yes, it is needed to be like that to allow several function to work on one research. meant to be used in refactoring later

File: maps/biter_battles_v2/functions.lua (hunk 2 above)
Comment (derkachdaniil) [reply maps/biter_battles_v2/functions.lua:15]: for now refactoring can be revolved around working with the most often one value in one different function, i make it later, when we can see, if some values will be repeating at all

Comment (derkachdaniil) [reply maps/biter_battles_v2/functions.lua]: note commited

**Comment by Ragnarok77-factorio** [comment]
Old PR , closed (needs vote approval anyways)

**Pull request closed without merging**